### DISCUSSION TITLE
how not to generate cells using jx:each ?

### DISCUSSION THREAD

**Discussion by yanxu217** [jxlsteam/jxls#187] [Q&A] [answered]
version: 2.11.0
excel template:
![image](https://user-images.githubusercontent.com/26690619/161223172-5d605300-9f93-44bc-9bdc-7f60c4c09ddc.png)
result:
![image](https://user-images.githubusercontent.com/26690619/161223557-07802073-c6ca-423d-8fae-a179525de390.png)
i want:
![image](https://user-images.githubusercontent.com/26690619/161223911-c8126ce2-5087-4435-b10d-9f7338bed766.png)

**Comment by SoltauFintel** [comment, marked as answer]
Looks like a fix sized invoice to me.

I think jxls 2.x does not support this.

You could add another jx:each (first cell: A4, lastCell="C4") in B4 and add as many empty elements as needed so the content is in the wanted row.

**Reply by Deninc** [reply]
hi @SoltauFintel 

> You could add another jx:each (first cell: A4, lastCell="C4") in B4 and add as many empty elements as needed so the content is in the wanted row.

I failed to understand how you could achieve the intended behavior of OP. Could you please give an example in Excel?
